src/Agent.Worker/Release/ReleaseDefinitionToFolderMap.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
+{
+    public sealed class ReleaseDefinitionToFolderMap
+    {
+        [JsonProperty("release_artifactsdirectory")]
+        public string ArtifactsDirectory { get; set; }
```

Comment:
artifacts directory is acutally  _work/r/1. We store the binaries in /a and other artifact related information else where.
I am fine with any of these names. Let me know if you feel strongly if we should change.